server/handler/transactions.go

```diff
@@ -159,6 +159,78 @@ func (h *Handler) GetTransactions(c echo.Context) error {
 	})
 }
 
+func (h *Handler) GetAllMultisigTxns(c echo.Context) error {
```

Comment:
We have an existing API for this `/multisig/:address/txs`. The only modification needed is to make the transaction status filter optional.

Reply:
we have multi-sig  account address transactions, we need accounts of all multisig txns.